exercises/minesweeper/minesweeper_test.py

```diff
@@ -2,150 +2,91 @@
 
 Implementation note:
 The board function must validate its input and raise a
-ValueError with a meaningfull error message if the
+ValueError with a *meaningful* error message if the
 input turns out to be malformed.
 """
 
 import unittest
 
-from minesweeper import board
+from minesweeper import annotate
 
 
 # Tests adapted from `problem-specifications//canonical-data.json` @ v1.1.0
 
 class MinesweeperTest(unittest.TestCase):
 
     def test_no_rows(self):
-        self.assertEqual(board([]), [])
+        self.assertEqual(annotate([]), [])
 
     def test_no_columns(self):
-        self.assertEqual(board([""]), [""])
+        self.assertEqual(annotate([""]), [""])
 
     def test_no_mines(self):
-        inp = ["   ",
-               "   ",
-               "   "]
-        out = ["   ",
-               "   ",
-               "   "]
-        self.assertEqual(board(inp), out)
-
-    def test_board_with_only_mines(self):
-        inp = ["***",
-               "***",
-               "***"]
-        out = ["***",
-               "***",
-               "***"]
-        self.assertEqual(board(inp), out)
+        minefield = ["   ", "   ", "   "]
+        out = ["   ", "   ", "   "]
+        self.assertEqual(annotate(minefield), out)
+
+    def test_annotate_with_only_mines(self):
+        minefield = ["***", "***", "***"]
+        out = ["***", "***", "***"]
+        self.assertEqual(annotate(minefield), out)
 
     def test_mine_surrounded_by_spaces(self):
-        inp = ["   ",
-               " * ",
-               "   "]
-        out = ["111",
-               "1*1",
-               "111"]
-        self.assertEqual(board(inp), out)
+        minefield = ["   ", " * ", "   "]
+        out = ["111", "1*1", "111"]
+        self.assertEqual(annotate(minefield), out)
 
     def test_space_surrounded_by_mines(self):
-        inp = ["***",
-               "* *",
-               "***"]
```

Comment:
Travis flake8 has never complained about vertical spacing. Please check your local configuration; it appears to be enforcing optional rules.

In this case, I would prefer to keep the vertical spacing on these tests as it improves readability of the input/expected output.

Reply:
Yup.  It would appear that I have it set to super-cranky.  I'll take another look at it's setup.  But on review, I also think it was complaining about the _alignment_ of those blocks (the indents didn't line up exactly) and not the fact that the line was broken up like that.  I'll correct and re-push.